docs/admin/upgrade.md

```diff
@@ -22,10 +22,11 @@ curl -L https://coder.com/install.sh | sh
 ```
 
 The script will unpack the new `coder` binary version over the one currently installed.
-Next, you can restart Coder with the following command (if running it as a system
+Next, you can restart Coder with the following commands (if running it as a system
 service):
 
 ```console
+systemctl daemon-reload
```

Comment:
Why is daemon-reload required?

Reply:
If upgrading `coder` via the curl to bash.